server/middlewares/stats.js

```diff
@@ -4,8 +4,11 @@ const client = redis.createClient(process.env.REDIS_URL)
 const mung = require('express-mung')
 const normalizeUrl = require('normalize-url')
 const cbor = require('borc')
+const ua = require('universal-analytics')
 
 const knownIps = new Set()
+const adaliteTrackingID = 'UA-143401606-1'
```

Comment:
this should not be hardcoded but provided from .env, see how other environment variables are provided. This property should be optional and if no "adaliteTrackingID" is defined, the middleware should not try to submit anything anywhere

Reply:
perhaps that's google analytics default behavior if the tracking id is undefined